public/maintenance.html

```diff
@@ -1,32 +1,92 @@
+            <p class="fs-2 text-muted"> Go ahead and take a break, you deserve it! </p>
+            <p class="fs-3 text-muted"> Don't worry, your projects, analyses, and data will be here for you when we come
+                back online.</p>
+            <div class="d-grid gap-2 d-md-block mx-auto my-5">
+                <a class="btn btn-lg" style="background-color:#0971ab;color:#ffff;" href="https://status.cyverse.org/"
+                   target="_blank">
+                    View Status of All Services
+                </a>
+                <a class="btn btn-lg" style="background-color:#0971ab;color:#ffff;"
+                   href="https://cyverse.org/maintenance" target="_blank">
+                    View Maintenance Calendar
+                </a>
+                <a class="btn btn-lg" style="background-color:#0971ab;color:#ffff;"
+                   href="https://user.cyverse.org/account" target="_blank">
+                    Enable Maintenance Reminders
+                </a>
+            </div>
         </div>
     </div>
 </div>
+<div class="center mt">
+    <a href="https://user.cyverse.org" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> Manage My Account</button>
+    </a>
+    <a href="https://de.cyverse.org" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> Discovery Environment</button>
+    </a>
+    <a href="https://cyverse.org/data-store" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> Data Store</button>
+    </a>
+    <a href="https://bisque.cyverse.org" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> BisQue Image Analysis</button>
+    </a>
+    <a href="https://dnasubway.cyverse.org" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> DNA Subway</button>
+    </a>
+    <a href="https://cyverse.org/Science-APIs" target="_blank">
+        <button type="button" class="cyverse-blue bottomButton"> Science APIs</button>
+    </a>
+</div>
+<div class="center mt">
+    <a href="https://twitter.com/CyVerseOrg" target="_blank"><img
+        src="https://cyverse.org/sites/default/files/inline-images/social_icon-01.png" class="icon" alt="twitter" /></a>
+    <a href="https://www.facebook.com/CyVerse.org/" target="_blank"> <img
+        src="https://cyverse.org/sites/default/files/inline-images/social_icon-02.png" class="icon"
+        alt="facebook" /></a>
+    <a href="https://www.youtube.com/channel/UC-gvdjTz9rq6RovZ57LoDDA/featured" target="_blank"><img
+        src="https://cyverse.org/sites/default/files/inline-images/social_icon-03.png" class="icon" alt="youtube" /></a>
+    <a href="https://www.linkedin.com/company/cyverse.org/" target="_blank"> <img
+        src="https://cyverse.org/sites/default/files/inline-images/social_icon-04.png" class="icon"
+        alt="linkedin" /></a>
+    <a href="https://github.com/cyverse/" target="_blank"> <img
```

Comment:
All of these links that use the `target="_blank"` attribute should also set a `rel="noopener noreferrer"` attribute (see https://mathiasbynens.github.io/rel-noopener/ for example).

All `target="_blank"` links could include this safety measure, though it's probably not as crucial to include `noreferrer` in links to other cyverse.org pages.